band/test/cpu_test.cc

```diff
@@ -28,20 +29,25 @@ namespace test {
 // NOTE: set may be different from kAll due to device-specific limitation
 // e.g., Galaxy S20 can only set affinity to first 6 cores
 TEST(CPUTest, AffinitySetTest) {
-  std::vector<CPUMaskFlag> masks = {CPUMaskFlag::kAll, CPUMaskFlag::kLittle,
-                                    CPUMaskFlag::kBig, CPUMaskFlag::kPrimary};
-  for (auto mask : masks) {
-    CpuSet target_set = BandCPUMaskGetSet(mask);
-    // this fails if target_set is null
-    absl::Status set_status = SetCPUThreadAffinity(target_set);
-    if (!set_status.ok()) {
-      EXPECT_EQ(target_set.NumEnabled(), 0);
-    } else {
-      sleep(1);
-      CpuSet current_set;
-      // should always success
-      EXPECT_EQ(GetCPUThreadAffinity(current_set), absl::OkStatus());
-      EXPECT_EQ(target_set, current_set);
+  if (!device::IsRooted()) {
+      std::cout << "As this device is unrooted, Cpu Affinity can't be set.\n" << std::endl;
```

Comment:
```suggestion
      GTEST_SKIP() << "As this device is unrooted, Cpu Affinity can't be set.\n" << std::endl;
```
Better solution : https://google.github.io/googletest/advanced.html#skipping-test-execution.

Reply:
JFYI, it's not good to print something during the test.

Reply:
I'll update as not printing something.